www/src/routes/ResumeReview.tsx

```diff
@@ -1,8 +1,31 @@
+import React, { FC, useEffect } from 'react';
+import axios from 'axios';
 import { useAuth0 } from '@auth0/auth0-react';
-import React, { FC } from 'react';
+import config from '../util/config';
 
 const ResumeReview: FC = () => {
-    const { isAuthenticated } = useAuth0();
+    const { isAuthenticated, getAccessTokenSilently } = useAuth0();
+
+    useEffect(() => {
```

Comment:
Maybe stick a `// TODO: Remove this` above this so we don't forget to take a random call to our api out of the code when we ship this.

Reply:
I think it might be best to just remove it entirely and cross this bridge again when we need to. This was just for the purposes of the demo.

Reply:
Yep, let's remove it then.